### PR TITLE
Feature/forced modal

### PR DESCRIPTION
**Pull request by raphaellueckl**
# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

File: src/components/30-organisms/modal/story.js [hunk 1]
```diff
@@ -18,10 +18,23 @@ export default {
...
+  const forced = boolean('forced', false);
 
   const wrapper = document.createElement('div');
+
+  setTimeout(() =>
+    document.querySelector('axa-button').addEventListener('click', () => {
+      document.querySelector('axa-modal').removeAttribute('open');
+    })
+  );
+
```
Comment (marcwelz) [inline src/components/30-organisms/modal/story.js:30]: Do you need `setTimeout()` is there no other way like we did on line 64?

Comment (raphaellueckl) [reply src/components/30-organisms/modal/story.js:30]: No that's not possible without using a native button (instead of our webcomponent).

**Review by marcwelz — DISMISSED** [review]

File: src/components/30-organisms/modal/README.md [hunk 2]
```diff
@@ -9,18 +9,25 @@ npm install @axa-ch/modal
...
 
 ### Variant
 
-| Attribute   | Details                   |
-| ----------- | ------------------------- |
-| `open=true` | State of modal visibility |
+| Attribute | Details                                          |
+| --------- | ------------------------------------------------ |
+| `open`    | State of modal visibility (boolean)              |
+| `forced`  | Disables the internal close mechanisms (boolean) |
```
Comment (danyball) [inline src/components/30-organisms/modal/README.md:25]: My first association with the word "forced" was, that this flag opens up the modal directly. Maybe you could find another word?

File: src/components/30-organisms/modal/index.scss [hunk 3]
```diff
@@ -65,6 +65,10 @@
     background-color: $color-prim-white;
     border-radius: 0px 0px 8px 8px;
 
+    &--forced {
+      border-radius: 8px 8px 8px 8px;
```
Comment (danyball) [inline src/components/30-organisms/modal/index.scss]: just `border-radius: 8px;` would be enough

File: src/components/30-organisms/modal/story.js [hunk 4]
```diff
@@ -56,6 +69,9 @@ export const Modal = () => {
         days, their intelligence quotient begins to decline. After three weeks
         hospitalization, it can already be 20% lower than usual.
       </axa-text>
+      <axa-button class="modal-story__close-modal-child-button">
+        I accept the terms and conditions
+      </axa-button>
```
Comment (danyball) [inline src/components/30-organisms/modal/story.js:74]: On my small 13" mac i did not see this button. Maybe its too much text on the modal for an example. minor.

File: src/components/30-organisms/modal/story.js [hunk 5]
```diff
@@ -56,6 +69,9 @@ export const Modal = () => {
...
     </axa-modal>
 
     <button
```
Comment (danyball) [inline src/components/30-organisms/modal/story.js:77]: do we need this button here for tests? because we have knobs and dont really need this button here.

**Review by danyball — CHANGES_REQUESTED** [review]

**Pull request merged**